README.md

```diff
@@ -135,6 +171,19 @@ $ dapr uninstall --all
 
 You should always run a `dapr uninstall` before running another `dapr init`.	
 
+#### Without Docker
+
+If Dapr has been installed in self-hosted mode with docker, run the following command to remove the installed binaries(daprd, placement): 
+
+```bash
+$ dapr uninstall --slim
```

Comment:
Does uninstall have to be separate, dapr unintall can take care of it with some checks in the code.

Reply:
it can but we will not be able to differentiate between not being able to connect to docker vs docker not being needed.

Reply:
Sure, we will now have 3 options:
dapr uninstall (removes dapr binaries and placement container)
dapr uninstall --all (removes dapr binaries, placement container, redis etc)
dapr uninstall --slim (removes dapr binaries.)

Reply:
Changed logic for uninstall and removed the `--slim` flag for uninstall.